bin/mock-user

```diff
@@ -8,7 +8,7 @@ import argparse
 
 
 def main(username, newsletter_consent_prompt=None):
-    from sentry.models.user import User
+    from sentry.users.models.users.user import User
```

Comment:
Could this be `sentry.users.models.user`? Or is the additional layer of modules helpful?

Reply:
Sure it definitely could go there. Just kinda worried that with all the other models, the models directory could grow p. large though we could do something like:
```
sentry.users.models
|
|--> auth
         |
         |authenticator.py
         
|--> emails
        
| user.py
| useremail.py
```
does this feel cleaner/makes more sense?

Reply:
nvm, read the rest of the review. 10 is p. few so I'm down to just flatten and put everything in models